pkg/cmd/container/cp_linux.go

```diff
@@ -39,17 +40,22 @@ func Cp(ctx context.Context, client *containerd.Client, options types.ContainerC
 				ctx,
 				client,
 				found.Container,
-				options.Container2Host,
-				options.DestPath,
-				options.SrcPath,
-				options.GOptions.Snapshotter,
-				options.FollowSymLink)
+				options)
 		},
 	}
 	count, err := walker.Walk(ctx, options.ContainerReq)
 
-	if count < 1 {
-		err = fmt.Errorf("could not find container: %s, with error: %w", options.ContainerReq, err)
+	if count == -1 {
+		if err == nil {
+			panic("nil error and count == -1 from ContainerWalker.Walk should never happen")
+		}
+		err = errors.Join(errors.New("unable to copy"), err)
```

Comment:
`fmt.Errorf` is fine

Reply:
I would say it is not, generally. It ends-up with wonky / un-uniform formatting across the codebase - making it hard for everybody where errors are coming from - inability to actually test against specific, typed-errors with errors.Is.
I am not going to fight for it here in this specific case (which is a bad hill to die on), but generally, I am strongly arguing that we should `Join` instead of `Errorf`.

Reply:
It is possible to error.Is by wrapping the error using the fmt pkg (we did some refacto in the past to switch nerdctl to fmt pkg)
The idea is:
-  to stick with current error formatting stragtegy.
- avoid import `errors` pkg

I think we can discuss the migration from fmt to errors.Join in a different PR

Reply:
Sure, you can error.Is if you have a typed error to compare with. Anyhow, will not fight it in this case.
Will amend when I get my key.

Reply:
Done.